wiki/Ranking_Criteria/Beatmap_Nominator_Rules/en.md

```diff
@@ -0,0 +1,56 @@
++ **Do not follow any unfinalized rules or amendments.** Only act upon the set of rules that is currently in effect, the rules of tomorrow are uncertain so they should not be followed.
++ **Make sure no mod posts in the map thread have been ignored before qualifying or bubbling a map.** Posts without a sufficient reply may count as ignored. When in doubt, double check if the Code of Conduct for Mapping and Modding has been violated on the mapthread and act accordingly.
++ **Ensure that kudosu is awarded to all posts that deserve it.** Additionally consult the mapper before giving out kudosu to make sure there are not any reasons why the post does not need kudosu (e.g. the modder has received one already for a different post which you overlooked). If the modpost is too old to receive kudosu, consult the modder to make an additional post linking the original he receives kudosu for.
++ **When nominating a Marathon mapset, always qualify the map first and then post the flame on the mapthread.** Pressing "Qualify" will automatically put a heart icon on the beatmap thread, and since the last icon is displayed posting the flame first will be overwritten by the heart. You are required to post a heart or flame in any case for logging reasons, though.
++ **Do not nominate a beatmap without properly checking all difficulties of the beatmap all by yourself.** If you are unfamiliar with the game mode some of the difficulties are mapped in, ask fellow Nominators for help. Improper checking includes just running a modding tool before nominating or not even downloading the beatmap. While difficult to prove, such behavior will be handled as inappropriate conduct.
++ **You can veto a beatmap set by placing a bubble pop on the set that you disagree with being fit for being Ranked.** Bubble popping over any kind of quality issues that you think need to be addressed before the map can move anywhere is one of your main responsibilities. A proper discussion with attempting to reach a mutual agreement by both sides of the argument has to follow.
++ **You are free to pop a rebubble of another nominator if you are not the original bubble popper and disagree with the reasoning for disregarding the bubble pop.** Thus the original popper is unable to overvote two other nominators.
++ **If the community member(s) requesting disqualification cannot reach an agreement with the mapper and you placed the last bubble or heart on a map, you cannot rebubble or requalify it.** For osu!taiko, osu!catch, osu!mania, and hybrid sets, you may break these conditions once per map due to the low number of nominators.
++ **Do not violate the terms of services or the osu! rules and the Code of Conduct for Modding and Mapping.** Misbehaving in public can and will get you expelled from the Beatmap Nomination Group.
```

Comment:
osu! must be italic here as it does not refer to the game mode itself but to the game as a whole (Wiki Style Guidelines). Call them osu! community rules. Link to osu! community rules (`/wiki/Rules`) and to the Code of Conduct (`/wiki/Ranking_Criteria/Code_of_Conduct`) when they are mentioned